src/kitovu/sync/plugin/moodle.py

```diff
@@ -98,7 +98,8 @@ def _list_files_in_course(self,
                                                        courseid=str(course_id))
 
         for section in lessons:
-            section_path: pathlib.PurePath = course_path / section['name']
+            section_nr = "{0:0=2d}".format(section['section'])
```

Comment:
I don't really understand the format string here - you can drop the initial `0` and just do `{:0=2d}` (no need to number fields in Python 2.7+). However, what's the `=` for? Looking at [pyformat.info](https://pyformat.info/#number_sign) it seems to control where the sign is going, but we probably don't expect negative numbers here? So I think this can just be `{:02d}` instead, right?

Reply:
Thanks for the clarification. TBH the documentation of the `format` method was too confusing for me to completely understand whilst writing this code. So I used this which seemed to work ;)